workspace-service/migrations/20201005095421_files.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS files (
+    id uuid DEFAULT uuid_generate_v4 (),
+    title TEXT NOT NULL,
+    description TEXT NOT NULL,
+    folder_id uuid NOT NULL REFERENCES workspaces,
```

Comment:
Should that reference `folders`?
```suggestion
    folder_id uuid NOT NULL REFERENCES folders,
```